dpgen/dispatcher/Slurm.py

```diff
@@ -163,7 +163,11 @@ def _check_status_inner(self, job_id):
                 raise RuntimeError\
                     ("status command squeue fails to execute\nerror message:%s\nreturn code %d\n" % (err_str, ret))
         status_line = stdout.read().decode('utf-8').split ('\n')[-2]
-        status_word = status_line.split ()[-4]
+        status_word = status_line.split ()[-1]
+        if not (len(status_line.split()) == 2 and status_word.isupper()): 
+            raise RuntimeError("Error in getting job status, " +
+                               "status_line = {status_line}, " + 
+                               "parsed status_word = {status_word}")
```

Comment:
Missing `f`?
```suggestion
                               f"parsed status_word = {status_word}")
```

Reply:
Yes. I forgot to add it when I attempted to format this line